tests/test_resources/test_clusters/test_cluster.py

```diff
@@ -1007,23 +1006,15 @@ def test_cluster_list_pythonic(self, cluster):
             token=rns_client.token,
             original_username=original_username,
         ):
-            clusters = Cluster.list()
-            all_clusters = clusters.get("den_clusters", {})
-            running_clusters = (
-                [
-                    den_cluster
-                    for den_cluster in all_clusters
-                    if den_cluster.get("Status") == "running"
-                ]
-                if all_clusters
-                else {}
-            )
-            assert len(all_clusters) > 0
-            assert len(running_clusters) > 0
-            assert len(running_clusters) == len(
-                all_clusters
+            all_clusters = Cluster.list(show_all=True).get("den_clusters", {})
```

Comment:
I think that this we should not remove test_cluster_list_pythonic (maybe just re-name it if necessary), because it tests the default behaviour of Cluster.list().

Seems to me we are going to show by default only part of the running clusters by default (based on @carolineechen comments on Josh's [PR](https://github.com/run-house/runhouse/pull/1431)), so IMO we need to have two separate tests:
1. Default behavior of Cluster.list()
2. Tests the behaviour of Cluster.list(show_all=true)

wdyt?

Reply:
yup makes sense, there was some overlap in my tests but separated to make it clearer. for the pythonic version, test for
* default behavior - contains only running clusters, includes current test cluster
* show_all - contains running + terminated clusters, includes current test cluster
* filter by running, terminated
* filter by time